README.md

```diff
@@ -1,10 +1,10 @@
 <img align="left" style="width:260px" src="https://github.com/raysan5/raylib/blob/master/logo/raylib_logo_animation.gif" width="288px">
 
-**raylib is a simple and easy-to-use library to enjoy videogames programming.**
+**raylib is a simple and easy-to-use library to enjoy videogame programming.**
```

Comment:
This is the official "slogan" for the library for +11 years, printed all around, I'm not changing it.